src/app.test.js

```diff
@@ -0,0 +1,17 @@
+const request = require('supertest');
+
+const app = require('./app');
+
+describe('GET /', () => {
+  it('Should respond with a message', () => {
+    const msg = 'Check out /contributors and /faq';
+    return request(app)
```

Comment:
I saw the issue in the stream based on not returning this promise chain. This has bitten me many times, and is generally a pretty solid argument to use async/await in tests. i.e.
```
it('Should respond with a message', async () => {
  const msg = 'Check out /contributors and /faq';
  const response = await request(app)
    .get('/')
    .expect('Content-Type', /json/)
    .expect(200);
  expect(response.body.msg).toBe(msg);
});
```